addons/website_event/static/src/snippets/options.js

```diff
@@ -62,7 +62,7 @@ options.registry.WebsiteEvent = options.Class.extend({
      * @private
      */
     _getEventObjectId() {
-        const objectIds = this.currentWebsiteUrl.match(/(?<=-)\d+(?![-\w\d])/);
-        return parseInt(objectIds[0]) | 0;
+        const objectIds = this.currentWebsiteUrl.match(/-\d+(?![-\w\d])/);
+        return parseInt(objectIds[0].replace(/-/, '')) | 0;
```

Comment:
Only difference is that regex replaces all occurrences and string replaces only the first occurrence. But here it doesn't matter since there should only be one '-' as per the regex, I just took a habit of using regex on replace.

I force pushed the change since I think it's clearer and more correct to use the string here.

Reply:
Not without the `g` flag, that's why I was confused
![image](https://user-images.githubusercontent.com/30048408/200027381-22ad1f6d-b4cf-470c-84e1-995f980f3cf8.png)